ui/js/signup.js

```diff
@@ -199,6 +202,13 @@ require(['jquery', 'underscore', 'oae.core'], function($, _, oae) {
             return;
         }
 
+        // Get the email domain for the tenant (if any) so we can validate the email
+        oae.api.admin.getTenant(null, function(err, tenant) {
```

Comment:
Is there a way in which we can avoid doing another API call? Can we return the email domain as part of the tenant object in the `me` feed?